src/app/views/kyb.go

```diff
@@ -137,23 +148,25 @@ func kybVerificationGroup(router *gin.Engine) {
 			return
 		}
 
-		c.JSON(http.StatusOK, gin.H{})
+		c.JSON(http.StatusOK, verification)
```

Comment:
no need to response verification on admin operators just {message: 'success'} would be enough
```suggestion
		c.JSON(http.StatusOK, gin.H{"message": "success"})
```